test/common/router/config_impl_test.cc

```diff
@@ -5050,6 +5050,24 @@ TEST_F(RouteMatcherTest, TestPrefixAndRegexRewrites) {
       "Cannot specify both prefix_rewrite and regex_rewrite");
 }
 
+TEST_F(RouteMatcherTest, TestPatternRewriteConfigLoad) {
```

Comment:
Does this actually test the functions you added in config_impl.cc?

Reply:
For now I am just testing config is loaded and fields are populated